plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/sku/style.scss

```diff
@@ -1,6 +1,11 @@
 .wc-block-components-product-sku {
 	display: block;
-	text-transform: uppercase;
 	@include font-size(small);
 	overflow-wrap: break-word;
+
+	strong {
+		text-transform: uppercase;
+		margin-left: $gap-smallest;
```

Comment:
This causes a 4px gap before SKU if the prefix is empty:

<img width="344" alt="image" src="https://github.com/user-attachments/assets/8dca3306-17be-49e6-a88b-4c2b193f23c7">